examples/cfd-dem/flow_in_bed.prm

```diff
@@ -50,17 +50,22 @@ end
 subsection void fraction
    set mode = dem
    set read dem = true
-   set dem file name = dem
-   set shock capturing = false
-   set reference velocity = 0.5
-   set grad div = true
-   set full stress tensor = false
+   set dem file name = packed_bed
+   set l2 smoothing factor = 0.000005
    set l2 lower bound = 0
    set l2 upper bound = 1
-   set l2 smoothing factor = 0.000005
-   set drag model = difelice
 end
 
+#---------------------------------------------------
+# CFD-DEM
+#---------------------------------------------------
+subsection cfd-dem
+   set grad div = true
+   set drag model = rong
```

Comment:
They are already present in the packed bed example in the CFD-DEM section of the Wiki

Reply:
But are they present in https://github.com/lethe-cfd/lethe/wiki/Understanding-the-Parameter-file-(.prm)  ?
Eventually it might be a good idea to create a page for the CFD-DEM parameters on the wiki :)

Reply:
done